### ISSUE TITLE
Editable installation use relative build-dir to rebuild

### ISSUE THREAD

**Issue by metab0t** [scikit-build/scikit-build-core#343]
When using an editable build with the following setting
```toml
[tool.scikit-build]
editable.rebuild = true
build-dir = "build/{wheel_tag}"
```

The import hook will use `build/{wheel_tag}` (the relative path) as `cwd` to run `cmake` and rebuild.
https://github.com/scikit-build/scikit-build-core/blob/9e2c6bd56c41b47b825c843bf6d1c6cc3aaf520b/src/scikit_build_core/resources/_editable_redirect.py#L75

If the current directory is not the package directory, the relative path provided to `cmake` will cause error.

**Comment by henryiii** [comment]
Should be fixed in #345. I'll be releasing pretty soon (working out one other bug), please let me know if it works!

**Comment by metab0t** [comment]
Thanks for the rapid response! The new version fixes this issue.
Closing this issue.
